docs/guides/integration/docker.md

```diff
@@ -340,18 +340,31 @@ WORKDIR /app
 RUN --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,source=uv.lock,target=uv.lock \
     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
-    uv sync --frozen --no-install-project
+    uv sync --frozen --no-install-project --link-mode=copy --compile-bytecode
```

Comment:
I think I have a preference for setting an environment variable for this as in https://github.com/astral-sh/uv-docker-example/pull/16

I think we cover bytecode compilation in a separate example though https://docs.astral.sh/uv/guides/integration/docker/#compiling-bytecode

Reply:
I agree that bytecode compilation shouldn't be included in this example, as it is already covered afterwards

Reply:
Fair point, although I feel obliged to point out that it also talk sabout caching later but uses a cache mount in this example already. :)

Reply:
I think I missed that when someone added the bind mounts to the example in #6921 — we should probably drop that from here too for clarity.